modules/rosa-getting-started-create-cluster-admin-user.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+[id="rosa-getting-started-create-cluster-admin-user_{context}"]
+= Creating a cluster admin user for quick cluster access
+
+Before configuring an identity provider, you can create a user with `cluster-admin` privileges for immediate access to your {product-title} (ROSA) cluster.
+
+[NOTE]
+====
+The cluster admin user is useful when you need quick access to a newly deployed cluster. However, Red Hat recommends that you configure an identity provider and grant cluster administrator privileges to the identity provider users as required. For more information about setting up an identity provider for your ROSA cluster, see _Configuring an identity provider and granting cluster access_.
```

Comment:
Thank you for highlighting this in the guidelines, that is useful to know. I have spent some time today trying to find an alternative way to word this without losing the sentiment. I think just removing "Red Hat recommends" implies that the note is saying not to use the `cluster-admin` user at all. Changing "Red Hat recommends" to something like, "for longer term use" or "outside of this scenario" is a bit too prescriptive, because there might be other scenarios where the cluster-admin user is useful.

In other words, this note is about a recommendation that is an alternative to the procedure in this section.

I will think on this for a while longer.

Reply:
@0kashi what is your view on this topic? Thanks.

Reply:
On reflection, I will leave this paragraph as is, because I think it is necessary to state that the reader can use the cluster admin role but that we recommend the use of the identity provider instead.